extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -465,27 +494,61 @@ export class AmpLightboxViewer extends AMP.BaseElement {
     this.updateInViewport(dev().assertElement(this.container_), true);
     this.scheduleLayout(dev().assertElement(this.container_));
 
-    this.currentElementId_ = element.lightboxItemId;
-    // Hack to access private property. Better than not getting
```

Comment:
I think it's better leave this comment or something similar to it. That way future readers of this code know it was an expedient hack. It might also be helpful to open a github issue to fix the hack here and add the issue number to the todo comment.

Reply:
Yup. In this case it's because we no longer need the `/**@type {?}*/ ` hack, since we changed the type of `this.carousel_` from `Element` to `CustomElement` and now type check no longer complains. 

```
this.carousel_.implementation_
     .showSlideWhenReady(this.currentElemId_);
```

Reply:
Actually I was wrong about this. I reverted my change for the time being, and will investigate how to avoid this hack.